SuperBuild/External_OpenCV.cmake

```diff
@@ -125,6 +120,7 @@ if(NOT DEFINED OpenCV_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
       -DCMAKE_DISABLE_FIND_PACKAGE_JAVA:BOOL=ON
       # Add Python wrapping, use Slicer's python
       -DOPENCV_SKIP_PYTHON_LOADER:BOOL=ON
+      -DOPENCV_PYTHON_EXTRA_DEFINITIONS:STRING=CV_RELEASE_PYTHON
```

Comment:
Almost, I think this should be `-DOPENCV_PYTHON_EXTRA_DEFINITIONS:BOOL=ON`

Reply:
No, see the changes [here](https://github.com/Slicer/opencv/commit/c8098f6c4001fd1a3a8f599f959beb3833f737b6)